base-template.sh

```diff
@@ -295,7 +295,7 @@ is_trusted_repo() {
 
         # shellcheck disable=SC2181
         if [ $TRUST_ALL_RESULT -ne 0 ]; then
-            echo "! This repository wants you to trust all current and future hooks without prompting"
+            echo "! This repository wants you to trust all current and future hooks without prompting" >&2
```

Comment:
This is a prompt rather than a failure, please keep it on stdout.

Reply:
The prompt comes afterwards I though (printf)